src/account/TransferTransaction.js

```diff
@@ -244,6 +249,24 @@ export default class TransferTransaction extends Transaction {
         return this;
     }
 
+    /**
+     * @internal
+     * @param {AccountId | string} accountId
+     * @param {number | string | Long | LongObject | BigNumber | Hbar} amount
+     * @returns {number | Long}
+     */
+    squashTransfers(accountId, amount) {
```

Comment:
This should have an `_` prefix. Also, I recommend you return `Hbar` instead of `number | Long`